Projekty/Projekt1/3.1_Kraszewski_Niewiadowski/Milestone3_OstateczneModele.ipynb

```diff
@@ -0,0 +1,1657 @@
+{
```

Comment:
<p>Tutaj warto wspomnieć, że <code>score</code>  to po prostu <code>accuracy</code>.  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>